client/src/views/SolutionPage.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="mj-solution-section">
+    <div v-if="mistake">
+      <div class="mj-solution-section-titles">{{ $t('Solutions.Name') }}</div>
+      <div class="mj-solution-section-items-content">
+        <fa-icon :icon="['far', 'star']" transform="grow-5"/>
+        <span class="mj-solution-section-items-text">
+          {{ mistake.name }}
+        </span>
+      </div>
+    </div>
+    <div v-if="mistake.goal">
+      <div class="mj-solution-section-titles">{{ $t('Solutions.Goal') }}</div>
+      <div class="mj-solution-section-items-content">
+        <fa-icon :icon="['far', 'dot-circle']" transform="grow-5"/>
+        <span class="mj-solution-section-items-text">
+          {{ mistake.goal }}
+        </span>
+      </div>
+    </div>
+    <div v-if="mistake.tips.length > 0 ">
+      <div class="mj-solution-section-titles">{{ $t('Solutions.Tips') }}</div>
+      <div
+        v-for="tip in mistake.tips"
+        :key="tip"
+        class="mj-solution-section-items-content">
+        <fa-icon :icon="['far', 'lightbulb']" transform="grow-5"/>
+        <span class="mj-solution-section-items-text">
+          {{ tip }}
+        </span>
+      </div>
+    </div>
```

Comment:
If there is no tips, I would consider some text like "No tips for a given mistake". I think that not showing a section might be misleading for user :) Plus, probably in the future we will need to add a functionality to add tips to mistake (and remove some old ones) so it would be useful to see a button "add tip" and this section.

Reply:
done :) But for now only showing text eq. no goal is set, because we don't know how Szymon would design these actions.